worldedit/primitives.lua

```diff
@@ -299,7 +299,6 @@ function worldedit.spiral(pos, length, height, spacer, node_name)
 		for index = 1, segment_length do
 			-- Move along the direction of the segment
 			i = i + stride_axis * sign
-			local column = i

```

Comment:
Incorrect.

Reply:
`column` was registered previously; this shouldn't be deleted.

Reply:
The value of `i` changes between the previous assignment and this one.